batch/batch/utils.py

```diff
@@ -248,7 +248,7 @@ def __init__(self):
         self._global_counter = WindowFractionCounter(10)
 
     def push(self, success: bool):
-        self._global_counter.push('exceeded_shares', success)
+        self._global_counter.push(secrets.token_urlsafe(6), success)
```

Comment:
what's this about?

Reply:
The interface for the `WindowFractionCounter` takes a key for idempotency. It was originally written for the case where we didn't want to count duplicate failures for the same operation ID. I'm reusing the code, but there isn't a "key" here. So I made up a random token as the key.